src/App/screens/Instructor/screens/LessonTopics/components/Submit/components/Well/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const typeSpecificClassNames = {
+  info: 'bg-light-blue',
+  error: 'bg-light-red',
+}
+
+export default ({
+  type = 'info',
+  description,
+}) => (
+  description
+    ? <div className={`pa3 br2 ${typeSpecificClassNames[type]}`}>
+        {description}
+      </div>
+    : null
+)
```

Comment:
This will most likely get promoted as a way to display relevant messages to the user throughout the app.